cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -201,6 +205,19 @@ This command should be used whenever a manual change in the migration directory
 		Args:    cobra.MaximumNArgs(1),
 		RunE:    CmdMigrateNewRun,
 	}
+	// MigrateSetRevisionCmd represents the 'atlas migrate set-revision' command.
+	MigrateSetRevisionCmd = &cobra.Command{
+		Use:   "set-revision <version> [flags]",
```

Comment:
If we set the revision, the argument should be called `revision`. But I would go with "version" instead of "revision".

Having said that, I don't like dashes in commands and I also think it's clear enough to read it this way:

```console
atlas migrate set <version>
```